app/src/main/res/layout/submitted_answer_item.xml

```diff
@@ -29,69 +29,85 @@
-      android:textSize="16sp"
-      android:visibility="gone" />
+      app:explorationSplitViewMarginApplicable="@{viewModel.hasConversationView &amp;&amp; viewModel.isSplitView}"
+      app:explorationSplitViewMarginBottom="@{@dimen/margin_0}"
+      app:explorationSplitViewMarginEnd="@{@dimen/margin_24}"
+      app:explorationSplitViewMarginStart="@{@dimen/margin_40}"
+      app:explorationSplitViewMarginTop="@{@dimen/margin_24}"
+      app:explorationViewMarginApplicable="@{viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
+      app:explorationViewMarginBottom="@{@dimen/margin_0}"
+      app:explorationViewMarginEnd="@{@dimen/margin_24}"
+      app:explorationViewMarginStart="@{@dimen/margin_40}"
+      app:explorationViewMarginTop="@{@dimen/margin_24}"
+      app:questionSplitViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; viewModel.isSplitView}"
+      app:questionSplitViewMarginBottom="@{@dimen/margin_0}"
+      app:questionSplitViewMarginEnd="@{@dimen/margin_32}"
+      app:questionSplitViewMarginStart="@{@dimen/margin_32}"
+      app:questionSplitViewMarginTop="@{@dimen/margin_24}"
+      app:questionViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
+      app:questionViewMarginBottom="@{@dimen/margin_0}"
+      app:questionViewMarginEnd="@{@dimen/margin_32}"
+      app:questionViewMarginStart="@{@dimen/margin_32}"
+      app:questionViewMarginTop="@{@dimen/margin_24}">
 
-    <androidx.recyclerview.widget.RecyclerView
-      android:id="@+id/submitted_answer_recycler_view"
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:background="@drawable/submitted_answer_background"
-      android:paddingStart="8dp"
-      android:paddingEnd="8dp"
-      android:visibility="gone"
-      app:itemDecorator="@{@drawable/divider}"
-      app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent"
-      app:list="@{submittedListAnswer.setOfHtmlStringsList}" />
+    <TextView
+        android:id="@+id/submitted_answer_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:background="@drawable/submitted_answer_background"
+        android:contentDescription="@{accessibleAnswer ?? submittedAnswer}"
+        android:paddingStart="12dp"
+        android:paddingTop="@{viewModel.hasConversationView ? @dimen/padding_16 : @dimen/padding_8}"
+        android:paddingEnd="12dp"
+        android:paddingBottom="@{viewModel.hasConversationView ? @dimen/padding_16 : @dimen/padding_8}"
+        android:textColor="@color/oppiaPrimaryText"
+        android:textSize="16sp"
+        android:visibility="gone"
+        app:layout_constrainedWidth="true"
+        android:text="@{submittedAnswer}"
+        app:layout_constraintEnd_toStartOf="@+id/submitted_answer_guideline"
```

Comment:
@MohamedMedhat1998 Can we do somethibg like -
```
app:layout_constraintEnd_toStartOf="@{viewModel.isCorrectAnswer ? @+id/submitted_answer_guideline : parent}"
```

Reply:
@MohamedMedhat1998 It is showing some error, can you help me in correcting the syntax?

Reply:
@prayutsu I think it can work. You will need to override `layout_constraintEnd_toStartOf` to work with data-binding. For reference, you can check [this file](https://github.com/oppia/oppia-android/blob/develop/app/src/main/java/org/oppia/android/app/databinding/GuidelineBindingAdapters.java) in which we override `layout_constraintGuide_percent`.

Reply:
Done